### PR TITLE
LGA-2580 -  Use new irsa service account

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

Use new irsa service account

## Any other changes that would benefit highlighting?

Remove old and unused template-deploy files (from when app was hosted in its own aws account)

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by psweeting1 — DISMISSED** [review]
LGTM

File: helm_deploy/cla-frontend/templates/_helpers.tpl [hunk 1]
```diff
@@ -102,3 +102,14 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of the service account to use
```
Comment (jogilder) [inline helm_deploy/cla-frontend/templates/_helpers.tpl:107]: Is this needed if create is always going to be false?

**Review by jogilder — CHANGES_REQUESTED** [review]
One comment but happy to approve with or without.

File: helm_deploy/cla-frontend/templates/_helpers.tpl (hunk 1 above)
Comment (said-moj) [reply helm_deploy/cla-frontend/templates/_helpers.tpl:107]: Removed the create flag

**Review by jogilder — APPROVED** [review]
LGTM

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**